src/modules/translation/defaults/en.json

```diff
@@ -1,7 +1,8 @@
 {
   "@dapps": {
     "button": {
-      "network_not_supported": "You need to be connected to {expectedChainName} to perform this action"
+      "network_not_supported": "You need to be connected to {expectedChainName} to perform this action",
```

Comment:
Would you mind building a container that injects these translations directly into the `BackToTopButton` like it's done in the [AssetsStatusFilter](https://github.com/decentraland/decentraland-dapps/blob/834b4f50bf4b05e7890824f67d9c84b9e7ea7600/src/containers/AssetStatusFilter/AssetStatusFilter.tsx#L9) component?